website/docs/r/pki_secret_backend_cert.html.md

```diff
@@ -84,3 +84,5 @@ In addition to the fields above, the following attributes are exported:
 * `serial_number` - The serial number
 
 * `expiration` - The expiration date of the certificate in unix epoch format
+
+* `renew_pending` - `true` if the current time (during refresh) is after the start of the early renewal window declared by `min_seconds_remaining`, and `false` otherwise; if `auto_renew` is set to `true` then the provider will plan to replace the certificate once renewal is pending.
```

Comment:
Would it be worth including an example of how to use this feature in a `postconditon` hook?

Reply:
For the moment I was intentionally trying to keep this doc change minimal because I believe the education team will be thinking about the overall strategy for documenting the use of postconditions in this way. Can we hold on this for the moment and then I would be happy to add an example here later if that matches the way the education team intends to document this? (We're expecting some similar changes in other providers, so I expect we'll want to have them all documented in a similar way but I don't know yet what that way will be.)

Reply:
SGTM!